apps/src/templates/feedback/LevelFeedbackEntry.jsx

```diff
@@ -138,6 +140,10 @@ export default class LevelFeedbackEntry extends Component {
     const showRightCaret = this.longComment() && !this.state.expanded;
     const showDownCaret = this.longComment() && this.state.expanded;
 
+    if (showRightCaret) {
+      firehoseClient.putRecord({study: 'all-feedback', event: 'long-comment'});
+    }
```

Comment:
I'm a little worried about logging a metrics event inside the `render()` method for this component.  This is something we probably want to do once per page load, but as written I think you'd send this every time you collapsed a comment too, or any time some other change caused a re-render.

We might get away with doing this in `componentDidMount` here - otherwise I'd want the parent component that owns the whole list to log this metric.

I'm also not sure this fully answers the question

> On the All Feedback page, how many comments are longer than the 2 lines shown initially?

This by itself will give us a count of how many long comments we've ever rendered, but the "how many" here might be more usefully interpreted as "How many visit" or even "What percentage per visit."  Either suggests we'd also want to log the total comments shown, which is probably easiest to do from the list's parent component.

Reply:
I want to know what % of comments are long -> long comments/total comments.

The weird part is that I'm considering the feedback long if the height it takes up is taller than 60px, so length is determined after the render. I'm not sure if the parent could (or should?) know length for an individual feedback in this scenario. 

Instead, I updated to also log the feedback id.  I think this will give us enough to answer the questions we're interested in. 

To find what % of comments are long you could query the database for the total number of teacher feedbacks and query Redshift for events == `long-feedback` and dedupe by feedback_id then count.

I'd still be sending more than I need, but now I have a way via feedback id to condense it down to the counts we're interested in. Is it expensive to store more in Redshift? Does this make sense?